packages/react-components/react-menu/library/src/components/MenuList/useMenuList.ts

```diff
@@ -150,8 +150,8 @@ export const useMenuList_unstable = (props: MenuListProps, ref: React.Ref<HTMLEl
       }),
       { elementType: 'div' },
     ),
-    hasIcons: menuContext.hasIcons || false,
-    hasCheckmarks: menuContext.hasCheckmarks || false,
+    hasIcons: menuContext.hasIcons || !!props.hasIcons,
+    hasCheckmarks: menuContext.hasCheckmarks || !!props.hasCheckmarks,
```

Comment:
Is this expected? Looks suspicious

Reply:
yeah previously - the menulist props were never used so you couldn't align the icons/checkmarks on a plain menu list https://stackblitz.com/edit/araqhz9v?file=src%2Fexample.tsx

Reply:
but I get that it doens't have to be a part of this PR - so I'll revert